### ISSUE TITLE
Hydra config not updated when RFdiffusion is run

### ISSUE THREAD

**Issue by jackoc893** [RosettaCommons/RFdiffusion#140]
Hi,

I have been trying to run the 'Basic execution - unconditional monomer' code on RFdiffusion. The code does run, but I do not get any output files. The outputted run_inference.txt file gives:

**[2023-10-12 17:22:49,396][rfdiffusion.inference.model_runners][INFO] - Reading checkpoint from C:\...\RF_Diffusion\RFdiffusion\rfdiffusion\inference/../../models/Base_ckpt.pt
[2023-10-12 17:22:51,870][rfdiffusion.inference.model_runners][INFO] - Loading checkpoint.
[2023-10-12 17:22:52,069][rfdiffusion.diffusion][INFO] - Using cached IGSO3.
[2023-10-12 17:22:52,088][__main__][INFO] - Making design samples/design_0
[2023-10-12 17:22:52,093][rfdiffusion.inference.model_runners][INFO] - Using contig: None**

And a manual inspection of the HYDRA config.yaml file reveals "contigs: null" in the contigmap sub-config. The following code was used:

**/scripts/run_inference.py 'contigmap.contigs=[150-150]' inference.output_prefix=test_outputs/test inference.num_designs=10**

So RFdiffusion does try to design the monomer, and the num_designs for the inference sub-config is updated for N=10, but it does not register any modifications to contigmap. Is there any workaround to this problem? Is it possible to run the RFdiffusion code with a manually edited config.yaml file (i.e. with the intended design specifications)?

Thank you in advance!
-J

**Comment by joewatchwell** [comment]
This is very surprising. Is this running on Windows? This is essentially just the example script, which runs fine on our systems.

**Comment by jackoc893** [comment]
Hi Joe,

Thank you for getting back to me - I have managed to resolve the issue! I did in fact run the initial code on a Windows OS (which had access to more GPU, so I had hoped to use the Linux subsystem for Windows as a workaround). I have now installed RFDiffusion on a Linux OS, and things seem to be running smoothly.

Thank you for your assistance (and for developing a software that actually does what it says on the tin!).

All the best,
-J
